core/src/main/java/org/ehcache/EhcacheManager.java

```diff
@@ -113,6 +114,17 @@ public EhcacheManager(Configuration config, Collection<Service> services, boolea
     this.configuration = new DefaultConfiguration(config);
   }
 
+
+  private void validateServicesConfigs() {
```

Comment:
If the build did not fail despite this, this means there is a test lacking somewhere. :)